From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Igor Pecovnik <igor.pecovnik@gmail.com>
Date: Mon, 21 Aug 2017 08:54:53 +0200
Subject: [ARCHEOLOGY] Increasing DMA block memory allocation to 2048k on all
 relevant kernels.

> X-Git-Archeology: > recovered message: > https://forum.armbian.com/index.php?/topic/4811-uas-mainline-kernel-coherent-pool-memory-size
> X-Git-Archeology: - Revision 908bb199ec2defd77f7f05d2016980abf100d627: https://github.com/armbian/build/commit/908bb199ec2defd77f7f05d2016980abf100d627
> X-Git-Archeology:   Date: Mon, 21 Aug 2017 08:54:53 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Increasing DMA block memory allocation to 2048k on all relevant kernels.
> X-Git-Archeology:
> X-Git-Archeology: - Revision a3cf7b74858e1f862db8ca238bd44f6406be6662: https://github.com/armbian/build/commit/a3cf7b74858e1f862db8ca238bd44f6406be6662
> X-Git-Archeology:   Date: Mon, 21 Aug 2017 17:52:29 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Revert "Increasing DMA block memory allocation to 2048k on all relevant kernels."
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2be21aad5dc965b3bc67e136a1e1170119d60f74: https://github.com/armbian/build/commit/2be21aad5dc965b3bc67e136a1e1170119d60f74
> X-Git-Archeology:   Date: Mon, 21 Aug 2017 17:52:48 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Increasing DMA block memory allocation to 2048k on all relevant kernels. https://forum.armbian.com/index.php?/topic/4811-uas-mainline-kernel-coherent-pool-memory-size
> X-Git-Archeology:
> X-Git-Archeology: - Revision b3d2bd4864d89ce032344051e6ced2ba9371084b: https://github.com/armbian/build/commit/b3d2bd4864d89ce032344051e6ced2ba9371084b
> X-Git-Archeology:   Date: Wed, 30 Aug 2017 05:42:08 +0000
> X-Git-Archeology:   From: Tonymac32 <tonymckahan@gmail.com>
> X-Git-Archeology:   Subject: Meson64 Dev increase DMA block memory allocation
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2c59bb9934b749b5df74d4134cd393dc24fd5160: https://github.com/armbian/build/commit/2c59bb9934b749b5df74d4134cd393dc24fd5160
> X-Git-Archeology:   Date: Mon, 18 Sep 2017 12:06:30 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Lepotato. Added NEXT, added patches from C2 NEXT, au, config update, removed deprecated patches, ...
> X-Git-Archeology:
> X-Git-Archeology: - Revision 13e94e58f04be27db51d18b7dac1d15a1864b79e: https://github.com/armbian/build/commit/13e94e58f04be27db51d18b7dac1d15a1864b79e
> X-Git-Archeology:   Date: Fri, 27 Oct 2017 16:14:21 +0300
> X-Git-Archeology:   From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
> X-Git-Archeology:   Subject: Remove random executable bits from patch files
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2c08ec8f5a210de35f9482f482ac01ea15381792: https://github.com/armbian/build/commit/2c08ec8f5a210de35f9482f482ac01ea15381792
> X-Git-Archeology:   Date: Thu, 24 May 2018 13:32:29 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Merge sunxi family into stable
> X-Git-Archeology:
> X-Git-Archeology: - Revision 7d2f3af08f23049c91c88eec5062613bbfbc85d4: https://github.com/armbian/build/commit/7d2f3af08f23049c91c88eec5062613bbfbc85d4
> X-Git-Archeology:   Date: Thu, 24 May 2018 15:44:15 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Merging Rockchip family
> X-Git-Archeology:
> X-Git-Archeology: - Revision 99a34c7be1e342247a981f99c7930ee73c144f3e: https://github.com/armbian/build/commit/99a34c7be1e342247a981f99c7930ee73c144f3e
> X-Git-Archeology:   Date: Tue, 26 Jun 2018 12:47:49 +0000
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Move Odroid C2 from 4.14.y to 4.16.y, added patch for ethernet, DEV to master, both tested ... not perfect but in a better condition.
> X-Git-Archeology:
> X-Git-Archeology: - Revision fcb85f17675990514d8fadc905e6ccc3bded7138: https://github.com/armbian/build/commit/fcb85f17675990514d8fadc905e6ccc3bded7138
> X-Git-Archeology:   Date: Thu, 28 Jun 2018 08:27:08 +0000
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Major Amlogic RFC and cleanup
> X-Git-Archeology:
> X-Git-Archeology: - Revision e261c6f82835bd9b12e07ba837b55fbf1aaa4327: https://github.com/armbian/build/commit/e261c6f82835bd9b12e07ba837b55fbf1aaa4327
> X-Git-Archeology:   Date: Wed, 31 Jul 2019 12:51:00 +0200
> X-Git-Archeology:   From: Aditya Prayoga <aprayoga@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move mvebu DEFAULT, NEXT and DEV branch to next kernel (LTS) and U-boot #1426 (#1487)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 150ac0c2afa147d9e3b036c8ecd8238fe5648cf3: https://github.com/armbian/build/commit/150ac0c2afa147d9e3b036c8ecd8238fe5648cf3
> X-Git-Archeology:   Date: Tue, 19 Nov 2019 23:25:39 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Remove K<4, change branches, new features (#1586)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0da16dfc3daacae374ece2778966297783ef004e: https://github.com/armbian/build/commit/0da16dfc3daacae374ece2778966297783ef004e
> X-Git-Archeology:   Date: Sun, 19 Jan 2020 15:54:02 +0100
> X-Git-Archeology:   From: Jannis <jannis@imserv.org>
> X-Git-Archeology:   Subject: [ mvebu ] Update patches to fit new legacy, current, dev
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2fe5e6f7553174c2b2be69d70398145e11af00b6: https://github.com/armbian/build/commit/2fe5e6f7553174c2b2be69d70398145e11af00b6
> X-Git-Archeology:   Date: Thu, 03 Dec 2020 10:24:27 +0100
> X-Git-Archeology:   From: Jannis <52237708+heisath@users.noreply.github.com>
> X-Git-Archeology:   Subject: [AR-558] Switch mvebu current to LK5.9 (based on previous mvebu-dev) (#2405)
> X-Git-Archeology:
> X-Git-Archeology: - Revision aa3d60f57e84d02887c63cae176bdec96b560e38: https://github.com/armbian/build/commit/aa3d60f57e84d02887c63cae176bdec96b560e38
> X-Git-Archeology:   Date: Thu, 10 Dec 2020 11:47:33 +0100
> X-Git-Archeology:   From: Rosen Penev <rosenp@gmail.com>
> X-Git-Archeology:   Subject: refreshed mvebu with quilt (#2419)
> X-Git-Archeology:
> X-Git-Archeology: - Revision bd4c2c67f07ec0bcd823332da00cc8d3a7d733d7: https://github.com/armbian/build/commit/bd4c2c67f07ec0bcd823332da00cc8d3a7d733d7
> X-Git-Archeology:   Date: Fri, 22 Jan 2021 13:20:44 +0100
> X-Git-Archeology:   From: Jannis <52237708+heisath@users.noreply.github.com>
> X-Git-Archeology:   Subject: [AR-609] Switch mvebu-current to 5.10.y (reusing config and patches from -dev) (#2547)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 5b1c1c2897a570c173c40204e98257b2dd7a74c9: https://github.com/armbian/build/commit/5b1c1c2897a570c173c40204e98257b2dd7a74c9
> X-Git-Archeology:   Date: Thu, 04 Jan 2024 00:06:37 +0530
> X-Git-Archeology:   From: Lane Jennison <lane@lane-fu.com>
> X-Git-Archeology:   Subject: mvebu-edge: move to 6.6.y
> X-Git-Archeology:
---
 arch/arm/mm/dma-mapping.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mm/dma-mapping.c b/arch/arm/mm/dma-mapping.c
index 111111111111..222222222222 100644
--- a/arch/arm/mm/dma-mapping.c
+++ b/arch/arm/mm/dma-mapping.c
@@ -182,7 +182,7 @@ static void *__alloc_remap_buffer(struct device *dev, size_t size, gfp_t gfp,
 				 pgprot_t prot, struct page **ret_page,
 				 const void *caller, bool want_vaddr);
 
-#define DEFAULT_DMA_COHERENT_POOL_SIZE	SZ_256K
+#define DEFAULT_DMA_COHERENT_POOL_SIZE	SZ_2M
 static struct gen_pool *atomic_pool __ro_after_init;
 
 static size_t atomic_pool_size __initdata = DEFAULT_DMA_COHERENT_POOL_SIZE;
-- 
Armbian

